File: examples/java-wav-file-example/src/main/java/org/example/SileroSpeechSegment.java
package org.example;


public class SileroSpeechSegment {
    private Integer startOffset;
    private Integer endOffset;
    private Float startSecond;
    private Float endSecond;

    public SileroSpeechSegment() {
    }

    public SileroSpeechSegment(Integer startOffset, Integer endOffset, Float startSecond, Float endSecond) {
        this.startOffset = startOffset;
        this.endOffset = endOffset;
        this.startSecond = startSecond;
        this.endSecond = endSecond;
    }

    public Integer getStartOffset() {
        return startOffset;
    }

    public Integer getEndOffset() {
        return endOffset;
    }

    public Float getStartSecond() {
        return startSecond;
    }

    public Float getEndSecond() {
        return endSecond;
    }

    public void setStartOffset(Integer startOffset) {
        this.startOffset = startOffset;
    }

    public void setEndOffset(Integer endOffset) {
        this.endOffset = endOffset;
    }

    public void setStartSecond(Float startSecond) {
        this.startSecond = startSecond;
    }

    public void setEndSecond(Float endSecond) {
        this.endSecond = endSecond;
    }
}

File: examples/csharp/SileroSpeechSegment.cs
namespace VadDotNet;

public class SileroSpeechSegment
{
    public int? StartOffset { get; set; }
    public int? EndOffset { get; set; }
    public float? StartSecond { get; set; }
    public float? EndSecond { get; set; }

    public SileroSpeechSegment()
    {
    }

    public SileroSpeechSegment(int startOffset, int? endOffset, float? startSecond, float? endSecond)
    {
        StartOffset = startOffset;
        EndOffset = endOffset;
        StartSecond = startSecond;
        EndSecond = endSecond;
    }
}